### PR TITLE
Mention IBC for TWS/Gateway

### PR DESCRIPTION
**Pull request by tredondo**
I mentioned IBeam at IBC as well.

https://github.com/IbcAlpha/IBC/issues/103#issuecomment-1698334336

### CODE REVIEW

**Comment by tredondo** [comment]
There's also a Docker image project with IB Gateway and IBC, https://github.com/UnusualAlpha/ib-gateway-docker

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ Features:
 * **Not so secure** - Yupp, you'll need to store the credentials somewhere, and that's a risk. Read more about it
   in [Security](#security).
 
+**NOTE**: IBeam is not designed to automate logging into TWS or IB Gateway. Use [IBC](IBC) for that.
```
Comment (Voyz) [inline README.md:34]: Could you change it to 

> ... or IB Gateway (also known as TWS Gateway)

as I heard some people calling it this way?

File: README.md [hunk 2]
```diff
@@ -242,3 +244,5 @@ Thanks and have an awesome day 👋
 [issues]: https://github.com/Voyz/ibeam/issues
 
 [gateway]: https://interactivebrokers.github.io/cpwebapi/
+
+[IBC]: https://github.com/IbcAlpha/IBC
```
Comment (Voyz) [inline README.md:248]: just for consistency, I use all-lowercase-with-dashes for link variables. Could you update it?

**Review by Voyz — CHANGES_REQUESTED** [review]
I appreciate this contribution @tredondo, good idea to mention it here 👏👏👏

**Comment by Voyz** [comment]
Great stuff @tredondo 💪 Thanks!

**Pull request merged**